fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -1102,10 +1102,13 @@ public PlanFragment visitPhysicalProject(PhysicalProject<? extends Plan> project
         }
 
         TupleDescriptor tupleDescriptor = generateTupleDesc(slotList, null, context);
-        inputPlanNode.setProjectList(execExprList);
+        if (!(inputPlanNode instanceof EmptySetNode)) {
+            inputPlanNode.setProjectList(execExprList);
+        }
```

Comment:
add a todo, we need to merge project(emptyset) to a new emptyset in rewrite step of Nereids